rootfs/scheduler/__init__.py

```diff
@@ -1004,12 +1004,7 @@ def _scale_rc(self, namespace, name, desired):
             'version': rc['spec']['selector']['version']
         }
 
-        # Are there any pods running (and verified as ready) available?
-        pods = self._get_pods(namespace, labels=labels).json()['items']
-        current = 0
-        for pod in pods:
-            if self._pod_ready(pod):
-                current += 1
+        current = int(rc['spec']['replicas'])
```

Comment:
the issue is happening at https://github.com/deis/controller/blob/master/rootfs/scheduler/__init__.py#L1015 which happens before the code we are setting the desired and updating the rc
here the issue is getting the value of the current before doing an update